std/json.d

```diff
@@ -166,7 +165,8 @@ struct JSONValue
     }
 
     /// Value getter/setter for $(D JSON_TYPE.STRING).
-    /// Throws $(D JSONException) for read access if $(D type) is not $(D JSON_TYPE.STRING).
+    /// Throws $(D JSONException) for read access if $(D type) is not
```

Comment:
All of these should be `Throws:` instead of just `Throws`. The colon on the end lets DDoc know that this is a separate section.